index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>kadai①-2</title>
+</head>
+<body>
+
+<!-- 課題はこちらのコードです -->
+ <script>
+
+let num = 1;
+const answer = '東京';
+
+  while (num < 5) {
+    var quiz = prompt('日本の首都は？');
+    if (quiz === answer) {
+      alert('正解');
+      num += 1;
```

Comment:
以下のように書く方がより一般的かなと思います。

```suggestion
      num++;
```